Makefile

```diff
@@ -44,8 +44,8 @@ unit-test:
 	go test $(GO_FILES) -v
 
 
-KIND_VERSION ?= 0.5.1
-KUBERNETES_VERSION ?= 1.15.3
+KIND_VERSION ?= 0.6.0
+KUBERNETES_VERSION ?= 1.16.3
```

Comment:
Since the driver is still expected to work with 1.15 and feature gates enabled, I think we should add a test in the pipeline to do 1.15 clusters and 1.16 clusters. We can add a presubmit test in the driver to do that. WDYT?

Reply:
> I think we should add a test in the pipeline to do 1.15 clusters and 1.16 clusters. We can add a presubmit test in the driver to do that.

SGTM. 

I was thinking we can update this e2e to 1.16+ as we don't have anything specific to 1.15 for the providers. WDYT?

Reply:
Yeah, that definitely makes sense. I'll get a presubmit pipeline setup for the driver. 

+1 for updating to 1.16+